usr/src/tools/env/common.sh

```diff
@@ -146,7 +122,20 @@ export MULTI_PROTO="no"
 # when the release slips (nah) or you move an environment file to a new
 # release
 #
-export VERSION="`git describe --long --all HEAD | cut -d/ -f2-`"
+export VERSION=$(git describe --long --all HEAD | cut -d/ -f2-)
+
+# Set the package version build number to a suitably granular timestamp, such
+# that it consistently increases, and is ideally newer than any distribution package.
+export PKGVERS_MAJOR=$(date "+%-Y")
+export PKGVERS_MINOR=$(date "+%-m")
+export PKGVERS_MICRO=$(date "+%-d")
+export PKGVERS_PATCH=$(date "+%-H")
+
+# Can't have 0 in package versions.
```

Comment:
Yes.  I had worse code to deal with leading-0s, and forgot a plain 0 was fine when I was cleaning it up.   Thanks.